src/styles.css

```diff
@@ -0,0 +1,121 @@
+* {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  border: 1px solid black;
+  border-radius: 5px;
```

Comment:
I would set the width to 200 and box-sizing to border-box.

The border color in the figma mock is definitely not black